srcpkgs/shake/template

```diff
@@ -12,11 +12,11 @@ homepage="http://vleu.net/shake/"
 distfiles="https://github.com/unbrice/$pkgname/archive/v$version.tar.gz"
 checksum=ac5a57bd8d55e197734852bf66318ecf39b4c5c298e43a259ba1d8a4b7a2ced3
 
-case "$XBPS_TARGET_MACHINE" in
-*-musl)
-	broken="musl does not provide error.h required by this package"
-	;;
-esac
+CFLAGS="-D_GNU_SOURCE"
```

Comment:
Is this line necessary? The build seems to complain a lot about redefining `_GNU_SOURCE`.